BUILD

```diff
@@ -0,0 +1,47 @@
+load("@bazel_gazelle//:def.bzl", "gazelle")
+load("@io_bazel_rules_go//go:def.bzl", "go_binary", "go_library")
+
+gazelle(
+    name = "gazelle",
+    prefix = "github.com/bazel-cache-s3",
```

Comment:
This should probably be `github.com/thomaso-mirodin/bazel-cache-s3` so as to mirror the actual import path

Reply:
Fixed